src/CodeSandbox.js

```diff
@@ -12,5 +12,5 @@ export const shouldTransform = url => {
 export const getHTML = url => {
   const iframeUrl = url.replace('/s/', '/embed/');
 
-  return `<iframe src="${iframeUrl}" style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"></iframe>`;
+  return `<iframe src="${iframeUrl}" style="width:100%; height:400px; border:0; border-radius: 4px; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe>`;
```

Comment:
```suggestion
  return `<iframe src="${iframeUrl}" style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe>`;
```

Don't forget to update the test snapshot too 😉